crates/c-api/include/wasmtime.h

```diff
@@ -833,6 +836,60 @@ WASM_API_EXTERN wasmtime_error_t *wasmtime_funcref_table_grow(
     wasm_table_size_t *prev_size
 );
 
+/**
+ * \brief Create a new `externref` value.
+ *
+ * Creates a new `externref` value wrapping the provided data.
+ *
+ * This function does not take an associated finalizer to clean up the data when
+ * the reference is reclaimed. If you need a finalizer to clean up the data,
+ * then use #wasmtime_externref_new_with_finalizer.
+ */
+WASM_API_EXTERN wasm_val_t wasmtime_externref_new(void *data);
```

Comment:
For this I believe the C API avoids returning structs-by-value where possible, so could the `wasm_val_t` be an out-pointer?